model/config.go

```diff
@@ -2326,7 +2329,15 @@ func (ss *ServiceSettings) isValid() *AppError {
 		}
 	}
 
-	if len(*ss.ListenAddress) == 0 {
+	host, port, err := net.SplitHostPort(*ss.ListenAddress)
```

Comment:
You are not handling the error here

Reply:
@jespino It is handling it down below once because they'll both return the same `AppError` so no need to do two returns.